deploy/kubernetes/filebeat-kubernetes.yaml

```diff
@@ -11,6 +11,8 @@ data:
     filebeat.inputs:
     - type: container
       paths:
+        #Uncomment below line if you want to enable CRI-O based logs autodiscover
+        #- /var/log/pods/${data.kubernetes.pod.uid}/${data.kubernetes.container.name}/*.log      
```

Comment:
Content like `${data.kubernetes.*}` is only available in configurations generated by the `kubernetes` provider, template based or hint's based. Here the configuration is the `container` input so the path should be sth like `/var/log/pods/*/*/*.log`. Haven't test it so it should be safe to make a test to ensure it's working.

Reply:
is there any issue with `/var/log/containers/*.log` when used cri-o?
`/var/log/containers/*.log` is usually just a symlink of `/var/log/pods/*/*/*.log`

Reply:
Let me test it.

If that is the case @tetianakravchenko then we dont need to do any changes at all. Not sure I was only following what we have in the code. Let me first test `/var/log/pods/*/*/*.log` and we can sync

Reply:
@tetianakravchenko You are correct, symlink it is. 

So removed it from here, I like better the /var/log/containers/*.log as have no subfolders.

Reply:
@gizas do we then need this PR if the default path `/var/log/containers/*.log` should work for different runtimes, including cri-o?

Reply:
Also /var/log/containers/*.log is symlink in containerd envs, checked it in docker.

On one hand our examples in doc show /var/log/pods/ and /var/lib/docker in each case. So we need to make sure that users have this info seperate per environment if they want to use actual links

Now the /var/log/containers/*.log is what we will propose in general, correct but I am not 100% sure that symlink is always the case in all envs. Just to be on safe side. 

Does it make sense?

Reply:
> On one hand our examples in doc show /var/log/pods/ and /var/lib/docker in each case

you mean `/var/log/containers/`, instead of `/var/log/pods/`?

> Now the /var/log/containers/*.log is what we will propose in general, correct but I am not 100% sure that symlink is always the case in all envs. Just to be on safe side.
Does it make sense?

to be on safe side - fine, but in this PR is emphasized that for cri-o runtime must be used `/var/log/pods/*` path, that is not correct and is misleading, no?

Reply:
@tetianakravchenko updated based on our discussion. Let me know if ok now?